src/content/capture/canvas-pc/js/test.js

```diff
@@ -22,6 +22,9 @@ test('PeerConnection pc1 sample', function(t) {
       '/src/content/peerconnection/pc1/index.html')
   .then(function() {
     t.pass('page loaded');
+    // Override the trace function to ensure console logging works for
+    // webdriver.
+    seleniumHelpers.overrideTrace(driver);
```

Comment:
I thought about that but wanted the test to explicitly do it in case we want the original trace logging but I guess that's something we could address with a toggle method later if needed in buildDriver. Will do as you suggest.